src/openms/include/OpenMS/APPLICATIONS/TOPPBase.h

```diff
@@ -524,12 +524,33 @@ namespace OpenMS
     */
     void registerOutputFile_(const String& name, const String& argument, const String& default_value, const String& description, bool required = true, bool advanced = false);
 
+    /**
+      @brief Registers an output file prefix used for tools with multiple file output.
+
+      Tools should follow the convention to name output files PREFIX_[0..N-1].EXTENSION.
+      For example, a tool that splits mzML files into multiple mgf files should create files:
+      splitted_0.mgf, splitted_1.mgf, ... if splitted got passed as prefix.
+      
+      Note: setting format(s) via setValidFormat_ for an output prefix will can used to export
+            e.g. valid CTD files that contain information on the expected output file types.            
```

Comment:
```suggestion
            e.g. valid CTD files that contain information on the expected output file types. In theory, it is possible
            to output different types and list them here but this should be avoided for cleanlyness (prefer multiple
            separate outputs). This could be left empty in case of an unknown amount of different extensions that
            are produced but is highly recommended.
```